src/audio/mixin_mixout.c

```diff
@@ -542,50 +710,70 @@ static int mixin_copy(struct comp_dev *dev)
 
 		sink_c = buffer_acquire(sink);
 
-		stream_free_bytes = audio_stream_get_free_bytes(&sink_c->stream);
+		/* Normally this should never happen as we checked above
```

Comment:
I'm not a native English speaker but "normally this should never happen" makes me slightly nervous and makes me think about parallel universes and alternative realities :-)

Reply:
Not sure what to do about this. Hope leaving as is is OK. Google knows 13600 occurrences of "normally this should never happen" with 164 of then on GitHub. Let it be one more :)